packages/css-library/src/tokens/line-height.scss

```diff
@@ -0,0 +1,8 @@
+$tokens-line-height: (
+  1: 1,
+  2: 1.15,
+  3: 1.35,
+  4: 1.5,
+  5: 1.62,
+  6: 1.75,
+);
```

Comment:
This seems a bit strange to me because I don't know what the numbers mean, but they [match what we already have](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/a831cc59059d2ef01c161f03e4fdb3caa5c5a8d4/packages/formation/sass/utilities/_line-height.scss#L5-L12). Adding a comment explaining what they mean might be nice.

We also may want these values to be tokens in StyleDictionary, but we can do that as a follow-up.